aten/src/ATen/native/nested/NestedTensorBinaryOps.cpp

```diff
@@ -146,9 +148,17 @@ Tensor NestedTensor_elementwise_Tensor(
   NestedTensorImpl* self_impl = nullptr;
   NestedTensorImpl* other_impl = nullptr;
   std::tie(self_impl, other_impl) =
-      get_elementwise_nested_tensor_impl(self, other, op_name);
+      get_elementwise_nested_tensor_impl(self, other, op_name, supports_striding);
   TORCH_INTERNAL_ASSERT_DEBUG_ONLY(self_impl);
   TORCH_INTERNAL_ASSERT_DEBUG_ONLY(other_impl);
+  if (supports_striding ) {
```

Comment:
Why not always do that? We know the code below will raise an error if this is not done right?

Reply:
You are asking the right question, and I dont' know why we don't do this? I imagine for because these calls to contiguous are very slow but idk